resources/devices/device_profile.json

```diff
@@ -0,0 +1,353 @@
+[
+  {
+    "step": 1,
```

Comment:
Can you please add fileds "title" and "description" on this level? 
Content: 
"title": "Select device type & technology", 
"description": "Before your device can go through testing, tell us more about your device and its functionality. It is important that we fully understand your device before a thorough assessment can be made."

Reply:
> "title": "Select device type & technology",
> "description": "Before your device can go through testing, tell us more about your device and its functionality. It is important that we fully understand your device before a thorough assessment can be made."

Done